src/components/product-option/product-option.tsx

```diff
@@ -0,0 +1,69 @@
+import { products } from '@wix/stores';
+import classNames from 'classnames';
+import styles from './product-option.module.scss';
+import { Select } from '../select/select';
+
+export interface ProductOptionProps {
+    option: products.ProductOption;
+    selectedValue: string | undefined;
+    onChange: (value: string) => void;
+}
+
+export const ProductOption = ({
+    option: { name, optionType, choices },
+    selectedValue,
+    onChange,
+}: ProductOptionProps) => {
+    if (name === undefined || choices === undefined) {
+        return undefined;
```

Comment:
pleas return null. it is so much more common practice. up to react18 it was even forbidden to return undefined from the render